actions/methods.go

```diff
@@ -9,13 +9,19 @@ import (
 	"github.com/mrz1836/go-datastore"
 )
 
-// RequestParameters is a struct for handling basic request parameters
-type RequestParameters struct {
+// SearchRequestParameters is a struct for handling request parameters for search requests
+type SearchRequestParameters struct {
 	Conditions  map[string]interface{} `json:"conditions"`
```

Comment:
It would be good to have description on all the request and response fields in swagger.